src/features/page/home/dialogs/LabeledInput.tsx

```diff
@@ -29,19 +35,25 @@ export default function LabeledInput({
   }));
 
   return (
-    <FormControlBlock>
+    <FormControlBlock error={error}>
       <InputLabel id="label1">{label}</InputLabel>
```

Comment:
moze sie juz troche przypierdalam ale no da sie chyba lepsze id zrobic niz label1 xD

Reply:
a byś sie przypierdolił w tamtym mr który to dodawał xD